### DISCUSSION TITLE
Sunbird-coKreat release 7.0.0

### DISCUSSION THREAD

**Discussion by Aparna-karri591** [Sunbird-coKreat/Community#98] [General Discussions]
Hi All,

Sunbird coKreat is going with the release-7.0.0 for which we will be using the following release notes from each of the building block 
@gandham-santhosh @santhosh-tg Please deploy the following release tags into dock staging.

|     Building Block          |  Release Note / tags |
|------------|:-------------|
| Sunbird Knowlg |  https://knowlg.sunbird.org/use/release-notes/release-5.7.0-latest | 
| Sunbird Inquiry |    https://inquiry.sunbird.org/use/release-notes/inquiry-release-v7.0.0-latest  | 
| Sunbird coKreat |   https://app.gitbook.com/o/-Mi9QwJlsfb7xuxTBc0J/s/y1wNXBlAD4R5UDMD45Vx/use/release-notes  | 

**Sunbird Knowlg** :  @mohitga @pallakartheekreddy

**Sunbird Inquiry** : @sajeshkayyath @PoojaJManjooran

Let us know if anything needs to be updated 

CC:
 @ashokreddy1208
@gandham-santhosh
@madhucr @alok-os
@anandp504 @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata  @gandham-santhosh @manju4201 @rajeevsathish @chitranshu-keshav @anisha147 @abhishekpnt @princegupta1131 @amitpriyadarshi @vinukumar-vs  @surendrasinghs @PoojaJManjooran  @omkar309 @abhay-samagra @shubhambansaltarento @vaivk369 @santhosh-tg

**Comment by shubhambansaltarento** [comment]
Release Tags are:
coKreat: [release-7.0.0_RC1](https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC1)
Program Service: [release-7.0.0_RC1](https://github.com/Sunbird-coKreat/program-service/releases/tag/release-7.0.0_RC1)
coKreate jobs - delete user flink Jobs: [release-7.0.0_RC1](https://github.com/Sunbird-coKreat/coKreat-jobs/releases/tag/release-7.0.0_RC1)

**Reply by vinukumar-vs** [reply]
@shubhambansaltarento Flik jobs tag is missing for delete user functionality.

[@gandham-santhosh](https://github.com/gandham-santhosh) [@PoojaJManjooran](https://github.com/PoojaJManjooran) @amitpriyadarshi

**Reply by shubhambansaltarento** [reply]
@vinukumar-vs flink job release tag created and added.
CC @ amitpriyadarshi

**Comment by sanketnagdive** [comment]
@shubhambansaltarento  CoKreat @Aparna-karri591 portal and other deployments are done, There are few pending deployments related flink jobs needs to be done. Will let you know once verificaton is done. cc @santhosh-tg @gandham-santhosh

**Reply by sanketnagdive** [reply]
@shubhambansaltarento @Aparna-karri591 we have swtich the domain to consume new environment where new release tags are deployed. Will work on onboarding of new api from each of the building blocks to be available in branch release-7.0.0-vdn also listed out all the challenges faced while deploying, will be updating here thread separetly cc @santhosh-tg @gandham-santhosh @vinukumar-vs @mohitga  @amitpriyadarshi @sajeshkayyath @PoojaJManjooran  @PoojaJManjooran @pallakartheekreddy

**Comment by shubhambansaltarento** [comment]
@sanketnagdive I have updated the Open Sabe jenkins job in release notes please deploy that job also for any other issue related to open saber please reffer this document [doc](https://project-sunbird.atlassian.net/wiki/spaces/SC/pages/3399254041/OpenSaber+aka+Contribution+Registry+in+coKreat)

**Comment by shubhambansaltarento** [comment]
@sanketnagdive please deploy the creation-portal with latest release tag -> [release-7.0.0_RC2](https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC2)

**Reply by sanketnagdive** [reply]
@shubhambansaltarento Its deployed to the dock staging env. cc @vinukumar-vs @santhosh-tg @gandham-santhosh

**Comment by shubhambansaltarento** [comment]
@sanketnagdive please deploy the creation-portal with latest release tag -> [release-7.0.0_RC3](https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC3)

**Comment by shubhambansaltarento** [comment]
@sanketnagdive please deploy the creation-portal with latest release tag -> [release-7.0.0_RC4](https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC4)

CC @vinukumar-vs @Aparna-karri591

**Reply by shubhambansaltarento** [reply]
@sanketnagdive also please deploy program service with latest release tag -> [release-7.0.0_RC2](https://github.com/Sunbird-coKreat/program-service/releases/tag/release-7.0.0_RC2)

**Comment by shubhambansaltarento** [comment]
@sanketnagdive -> please deploy creation portal and program service with latest build tags 

coKreat -> [release-7.0.0_RC5](https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC5)
programService -> [release-7.0.0_RC3](https://github.com/Sunbird-coKreat/program-service/releases/tag/release-7.0.0_RC3)

**Reply by sanketnagdive** [reply]
This is deployed on staging. cc @shubhambansaltarento @vinukumar-vs @santhosh-tg @gandham-santhosh

**Comment by shubhambansaltarento** [comment]
@sanketnagdive please deploy program-service with the latest releasetags -> [release-7.0.0_RC4](https://github.com/Sunbird-coKreat/program-service/releases/tag/release-7.0.0_RC4)

**Reply by sanketnagdive** [reply]
This is deployed @shubhambansaltarento

**Comment by shubhambansaltarento** [comment]
@sanketnagdive please deploy creation pooratl with the latest releae tag 
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC6
this has fix for CO-783 and CO-784

CC @vinukumar-vs

**Reply by santhosh-tg** [reply]
@shubhambansaltarento  Its deployed.

**Comment by shubhambansaltarento** [comment]
@sanketnagdive please deploy program - service with the latest release tag 
https://github.com/Sunbird-coKreat/program-service/releases/tag/release-7.0.0_RC5
also please revert default channel on coKreat from agriculure to sunbird and depploy cokreat also.

CC @vinukumar-vs

**Comment by shubhambansaltarento** [comment]
@sanketnagdive please deploy creation - portal (Player) with the latest release tag 
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC7

CC @vinukumar-vs

**Comment by shubhambansaltarento** [comment]
@sanketnagdive please deploy program-service with the latest release tag
https://github.com/Sunbird-coKreat/program-service/releases/tag/release-7.0.0_RC6

CC @vinukumar-vs

**Reply by sanketnagdive** [reply]
@shubhambansaltarento Its deployed cc @vinukumar-vs

**Comment by shubhambansaltarento** [comment]
@sanketnagdive can you please deploy program service with the latest release tag 
https://github.com/Sunbird-coKreat/program-service/releases/tag/release-7.0.0_RC8

CC @vinukumar-vs

**Comment by shubhambansaltarento** [comment]
@sanketnagdive can you please deploy program service with the latest release tag
https://github.com/Sunbird-coKreat/program-service/releases/tag/release-7.0.0_RC9

CC @vinukumar-vs

**Comment by shubhambansaltarento** [comment]
@sanketnagdive can you please deploy program service with the latest release tag
https://github.com/Sunbird-coKreat/program-service/releases/tag/release-7.0.0_RC10

CC @vinukumar-vs

**Reply by sanketnagdive** [reply]
@shubhambansaltarento release-7.0.0_RC10 is builded and deployed. cc @vinukumar-vs

**Comment by gandham-santhosh** [comment]
Request everyone to include the JIRA issue Id and release notes changes while requesting for deployment @shubhambansaltarento

**Comment by shubhambansaltarento** [comment]
@sanketnagdive can you please deploy program service with the latest release tag
https://github.com/Sunbird-coKreat/program-service/releases/tag/release-7.0.0_RC11

This is to test CO-800

CC @vinukumar-vs

**Comment by shubhambansaltarento** [comment]
@sanketnagdive can you please deploy creation portal with the latest release tag
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC8
This release should fix CO-803, CO-802, CO-791, CO-785

CC @vinukumar-vs

**Reply by shubhambansaltarento** [reply]
@sanketnagdive updated RC Tag for above issues 
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC9

**Comment by shubhambansaltarento** [comment]
@sanketnagdive please deploy creation portal with the latest release tag 
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC10
This tag should fix CO-800

CC @vinukumar-vs @Aparna-karri591

**Reply by santhosh-tg** [reply]
@shubhambansaltarento  Build and deployed to dock staging

**Comment by shubhambansaltarento** [comment]
@sanketnagdive please deploy creation portal with the latest release tag
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC11
This tag should fix CO-817

CC @vinukumar-vs @Aparna-karri591

**Comment by shubhambansaltarento** [comment]
@sanketnagdive please re-deploy the content editor job for coKreat and please rebuild and redeploy coKtreat portal with the latest path of content editor received after deploying it 

This will fix 7.0.0 blocker CO-817

CC @vinukumar-vs @Aparna-karri591

**Reply by sanketnagdive** [reply]
@shubhambansaltarento deployed the content editor, but not sure which path needs to be used for the creation portal build and deployment. cc @santhosh-tg @gandham-santhosh @vinukumar-vs @Aparna-karri591

**Comment by Aparna-karri591** [comment]
**R 7.0.0 Sunbird CoKreat - Development Phase Conditional Sign Off**

Hello All,

This is Development Phase QA Conditional Sign off for the release 7.0.0 Sunbird CoKreat Building block.

**Environment:** Dock Staging
**Testing Status:**  98.32 % Completed 
**Defects Dashboard:** [Link](https://project-sunbird.atlassian.net/jira/dashboards/10652)
**Execution Sheet**:[Link](https://docs.google.com/spreadsheets/d/1Wkq73Z3E7vZ8DK8J9o1GOAV4V0CmtdG97btzueEuVW8/edit#gid=32578273)
**Scope:** [Link](https://project-sunbird.atlassian.net/issues/?filter=12932)
 **Building Block Sign Off Template:** [Link](https://docs.google.com/spreadsheets/d/1FHRVnT8aeRDjDsCxPUFHQAr1X3Lkqr0pYulYgkT5MVk/edit#gid=171383707)

**Note:**

-  We are giving conditional Sign off since  we have 2 Open defects [[CO-760](https://project-sunbird.atlassian.net/browse/CO-760),[CO-812](https://project-sunbird.atlassian.net/browse/CO-812)] and Code Percentage is not called out.
- [CO-756](https://project-sunbird.atlassian.net/browse/CO-756)This Defect Converted Bug

Sunbird ED @mohitga @vinukumar-vs
Sunbird Knowlg @mohitga @pallakartheekreddy
Sunbird Inquiry [@PoojaJManjooran](https://github.com/PoojaJManjooran) @sajeshkayyath
Sunbird CoKreate @vinukumar-vs @PoojaJManjooran
Sunbird Lern @PoojaJManjooran @AmiableAnil
Sunbird Observ @rhwarrier @anandp504
Sunbird ED Devops 
[@Santhosh Gandham](https://ekstep.slack.com/team/U78TDJSD7)
 @sanketnagdive @santhosh-tg
Sunbird QA: @ashokreddy1208 @PoornimaRajesh

CC :
@madhucr @alok-os @vijiurs @Prateek-slokam @aks30 @manju4201 @rajeevsathish 
[@Chitranshu](https://ekstep.slack.com/team/UEXC3SL83)
 @anisha147 @abhishekpnt @princegupta1131 @amitpriyadarshi 
[@Swayangjit](https://ekstep.slack.com/team/U04DQSB0D)
 @kiranharidas187
@shubhambansaltarento

**Comment by ashokreddy1208** [comment]
Hi All,

Sunbird coKreat is going to start 7.0.0 Regression testing on staging env, for which we will be using the following release notes from each of the building block
@gandham-santhosh @santhosh-tg Please deploy the following release tags into dock staging.


Building Block | Release Note / tags
-- | --
Sunbird Knowlg | https://knowlg.sunbird.org/use/release-notes/release-5.7.0
Sunbird inQuiry | https://inquiry.sunbird.org/use/release-notes/inquiry-release-v7.0.0-latest
Sunbird coKreat | Sunbird-coKreat: release-7.0.0_RC12, Sunbird-programService: release-7.0.0_RC11



Sunbird ED @mohitga @vinukumar-vs
Sunbird Knowlg @mohitga @pallakartheekreddy
Sunbird Inquiry [@PoojaJManjooran](https://github.com/PoojaJManjooran) @sajeshkayyath
Sunbird CoKreate @vinukumar-vs @PoojaJManjooran
Sunbird Lern @PoojaJManjooran @AmiableAnil
Sunbird Observ @rhwarrier @anandp504
Sunbird ED Devops
[@Santhosh Gandham](https://ekstep.slack.com/team/U78TDJSD7)
@sanketnagdive @santhosh-tg
Sunbird QA: @ashokreddy1208 @PoornimaRajesh

CC :
@madhucr @alok-os @vijiurs @Prateek-slokam @aks30 @manju4201 @rajeevsathish
[@Chitranshu](https://ekstep.slack.com/team/UEXC3SL83)
@anisha147 @abhishekpnt @princegupta1131 @amitpriyadarshi
[@Swayangjit](https://ekstep.slack.com/team/U04DQSB0D)
@kiranharidas187
@shubhambansaltarento

**Comment by Aparna-karri591** [comment]
@gandham-santhosh @sanketnagdive @santhosh-tg can you please conform is coKreat 7.0 deployment done on dockstaging?

**Reply by sanketnagdive** [reply]
@Aparna-karri591 Cokreat deployment is still not done, will let you know once it is done. We are preparing sheet with the latest changes cc @santhosh-tg @gandham-santhosh

**Comment by sanketnagdive** [comment]
Cokreat deployment along with knowlg and inquiry has been deployed to staging environment by considering latest release notes. Please start validating.

Sunbird ED @mohitga @vinukumar-vs
Sunbird Knowlg @mohitga @pallakartheekreddy
Sunbird Inquiry [@PoojaJManjooran](https://github.com/PoojaJManjooran) @sajeshkayyath
Sunbird CoKreate @vinukumar-vs @PoojaJManjooran
Sunbird Lern @PoojaJManjooran @AmiableAnil
Sunbird Observ @rhwarrier @anandp504
Sunbird ED Devops
[@Santhosh Gandham](https://ekstep.slack.com/team/U78TDJSD7)
@sanketnagdive @santhosh-tg
Sunbird QA: @ashokreddy1208 @PoornimaRajesh

CC :
@madhucr @alok-os @vijiurs @Prateek-slokam @aks30 @manju4201 @rajeevsathish
[@Chitranshu](https://ekstep.slack.com/team/UEXC3SL83)
@anisha147 @abhishekpnt @princegupta1131 @amitpriyadarshi
[@Swayangjit](https://ekstep.slack.com/team/U04DQSB0D)
@kiranharidas187
@shubhambansaltarento

**Comment by shubhambansaltarento** [comment]
@sanketnagdive can you please re-deploy the coKreat creation portal using this release tag 
[release-7.0.0_RC13](https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC13)

This will fix -> CO-925

**Reply by sanketnagdive** [reply]
It is deployed @shubhambansaltarento

**Comment by Namratakushwah** [comment]
Hi All,

Please find the testing status of coKreat R7.0.0 as of 3rd May 2024

Environment: coKreat Dock Staging
Testing completion percentage: In Progress (68.75% Completed)
Planned Sign-Off Date: 6th May 2024
Extended Planned Sign-Off Date: 10th May 2024
Defects Dashboard: [[Link]](https://project-sunbird.atlassian.net/issues/?filter=12868)
Execution sheet: [[Link]](https://docs.google.com/spreadsheets/d/1-0UaycSNdLqlXCmllYJEsl7xKgxXfxVx4uJYZYorSwI/edit#gid=518610797)

**Planned Vs Actual Testing Status:**

![image](https://github.com/Sunbird-coKreat/Community/assets/129187143/ec613b8f-e8e2-4573-a176-cb9fcf70b63d)


CC:
@mohitga 
@ashokreddy1208
@gandham-santhosh
@madhucr @alok-os
 @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @aayushkedia95 @manju4201 @rajeevsathish @chitranshu-keshav @anisha147 @abhishekpnt @princegupta1131 @amitpriyadarshi @vinukumar-vs @surendrasinghs @PoojaJManjooran  @abhay-samagra @shubhambansaltarento @vaivk369 @santhosh-tg @PoornimaRajesh

**Comment by Namratakushwah** [comment]
Hi All,

Please find the testing status of coKreat R7.0.0 as of 6th May 2024

Environment: coKreat Dock Staging
Testing completion percentage: In Progress (73.73% Completed)
Planned Sign-Off Date: 6th May 2024
Extended Planned Sign-Off Date: 10th May 2024
Defects Dashboard: [[Link]](https://project-sunbird.atlassian.net/issues/?filter=12868)
Execution sheet: [[Link]](https://docs.google.com/spreadsheets/d/1-0UaycSNdLqlXCmllYJEsl7xKgxXfxVx4uJYZYorSwI/edit#gid=518610797)

**Planned Vs Actual Testing Status:**

![image](https://github.com/Sunbird-coKreat/Community/assets/129187143/e0bb5375-024c-4a39-9b20-a28970876251)

CC:
@mohitga
@ashokreddy1208
@gandham-santhosh
@madhucr @alok-os
@aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @aayushkedia95 @manju4201 @rajeevsathish @chitranshu-keshav @anisha147 @abhishekpnt @princegupta1131 @amitpriyadarshi @vinukumar-vs @surendrasinghs @PoojaJManjooran @abhay-samagra @shubhambansaltarento @vaivk369 @santhosh-tg @PoornimaRajesh

**Comment by Namratakushwah** [comment]
Hi All,

Please find the testing status of coKreat R7.0.0 as of 7th May 2024

Environment: coKreat Dock Staging
Testing completion percentage: In Progress (79.46% Completed)
Planned Sign-Off Date: 6th May 2024
Extended Planned Sign-Off Date: 10th May 2024
Defects Dashboard: [[Link]](https://project-sunbird.atlassian.net/issues/?filter=12868)
Execution sheet: [[Link]](https://docs.google.com/spreadsheets/d/1-0UaycSNdLqlXCmllYJEsl7xKgxXfxVx4uJYZYorSwI/edit#gid=518610797)

**Planned Vs Actual Testing Status:**
![image](https://github.com/Sunbird-coKreat/Community/assets/129187143/7b53cdc7-9c4f-4f8e-ad16-ca155aaf1104)

CC:
@mohitga
@ashokreddy1208
@gandham-santhosh
@madhucr @alok-os
@aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @aayushkedia95 @manju4201 @rajeevsathish @chitranshu-keshav @anisha147 @abhishekpnt @princegupta1131 @amitpriyadarshi @vinukumar-vs @surendrasinghs @PoojaJManjooran @abhay-samagra @shubhambansaltarento @vaivk369 @santhosh-tg @PoornimaRajesh

**Comment by Namratakushwah** [comment]
Hi All,

Please find the testing status of coKreat R7.0.0 as of 8th May 2024

Environment: coKreat Dock Staging
Testing completion percentage: In Progress (82.80% Completed)
Planned Sign-Off Date: 6th May 2024
Extended Planned Sign-Off Date: 10th May 2024
Defects Dashboard: [[Link]](https://project-sunbird.atlassian.net/issues/?filter=12868)
Execution sheet: [[Link]](https://docs.google.com/spreadsheets/d/1-0UaycSNdLqlXCmllYJEsl7xKgxXfxVx4uJYZYorSwI/edit#gid=518610797)

**Planned Vs Actual Testing Status:**

![image](https://github.com/Sunbird-coKreat/Community/assets/129187143/485beef3-784d-425d-867f-ee399938cb77)
CC:
@mohitga
@ashokreddy1208
@gandham-santhosh
@madhucr @alok-os
@aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @aayushkedia95 @manju4201 @rajeevsathish @chitranshu-keshav @anisha147 @abhishekpnt @princegupta1131 @amitpriyadarshi @vinukumar-vs @surendrasinghs @PoojaJManjooran @abhay-samagra @shubhambansaltarento @vaivk369 @santhosh-tg @PoornimaRajesh

**Comment by Namratakushwah** [comment]
Hi All,

Please find the testing status of coKreat R7.0.0 as of 9th May 2024

Environment: coKreat Dock Staging
Testing completion percentage: In Progress (89.37% Completed)
Planned Sign-Off Date: 6th May 2024
Extended Planned Sign-Off Date: 10th May 2024
Defects Dashboard: [[Link]](https://project-sunbird.atlassian.net/issues/?filter=12868)
Execution sheet: [[Link]](https://docs.google.com/spreadsheets/d/1-0UaycSNdLqlXCmllYJEsl7xKgxXfxVx4uJYZYorSwI/edit#gid=518610797)

**Planned Vs Actual Testing Status:**
![image](https://github.com/Sunbird-coKreat/Community/assets/129187143/8d9018cb-d952-4b05-bc30-f07837488b06)

CC:
@mohitga
@ashokreddy1208
@gandham-santhosh
@madhucr @alok-os
@aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @aayushkedia95 @manju4201 @rajeevsathish @chitranshu-keshav @anisha147 @abhishekpnt @princegupta1131 @amitpriyadarshi @vinukumar-vs @surendrasinghs @PoojaJManjooran @abhay-samagra @shubhambansaltarento @vaivk369 @santhosh-tg @PoornimaRajesh

**Comment by shubhambansaltarento** [comment]
@sanketnagdive can you please re-deploy coKreat and program service using following release tags 
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC14. -- coKreart
https://github.com/Sunbird-coKreat/program-service/releases/tag/release-7.0.0_RC12 -- program service

this will resolce CO-942, CO-923, CO-941

CC @vinukumar-vs @ashokreddy1208

**Comment by shubhambansaltarento** [comment]
@sanketnagdive can you please re-deploy coKreat and program service using following release tags
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC15 -- coKreart
https://github.com/Sunbird-coKreat/program-service/releases/tag/release-7.0.0_RC13 -- program service

this will resolce CO-942, CO-760, CO-951, CO-944

CC @vinukumar-vs @ashokreddy1208 @santhosh-tg @gandham-santhosh

**Reply by santhosh-tg** [reply]
@shubhambansaltarento  deployment done

**Comment by shubhambansaltarento** [comment]
@sanketnagdive  can you please deploy coKreat using following release tag
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC16

This will fix CO-957
CC @vinukumar-vs @ashokreddy1208 @santhosh-tg @gandham-santhosh

**Comment by shubhambansaltarento** [comment]
@sanketnagdive can you please deploy coKrear using following release tag 
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC17

this will fix, CO-923, CO941
CC @ashokreddy1208 @vinukumar-vs @santhosh-tg @gandham-santhosh

**Reply by sanketnagdive** [reply]
@shubhambansaltarento This is deployed

**Comment by shubhambansaltarento** [comment]
@sanketnagdive can you please re-deploy coKreat and program service using following release tags
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC18
https://github.com/Sunbird-coKreat/program-service/releases/tag/release-7.0.0_RC14

this will fix -> CO-961, 950
CC @ashokreddy1208 @vinukumar-vs @santhosh-tg @gandham-santhosh

**Reply by sanketnagdive** [reply]
@shubhambansaltarento This are deployed cc @ashokreddy1208 @vinukumar-vs @santhosh-tg @gandham-santhosh

**Comment by shubhambansaltarento** [comment]
@sanketnagdive can you please re-deploy coKreat creatioin portal using following release tags 
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC19

this will fix CO-946
CC @ashokreddy1208 @vinukumar-vs @santhosh-tg @gandham-santhosh

**Reply by sanketnagdive** [reply]
It is deployed CC @ashokreddy1208 @vinukumar-vs @santhosh-tg @gandham-santhosh @shubhambansaltarento

**Comment by mentorSprigBoard304** [comment]
@sanketnagdive can you please re-deploy coKreat creatioin portal using following release tags
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC20

this will fix CO-950
CC @ashokreddy1208 @vinukumar-vs @santhosh-tg @gandham-santhosh

**Reply by sanketnagdive** [reply]
it is deployed

**Comment by shubhambansaltarento** [comment]
@sanketnagdive can you please re-deploy coKreat creatioin portal using following release tags
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC20

this will fix CO-950
CC @ashokreddy1208 @vinukumar-vs @santhosh-tg @gandham-santhosh

**Reply by sanketnagdive** [reply]
@shubhambansaltarento seems this is same deployment as mentioned in earlier comment.

**Comment by shubhambansaltarento** [comment]
@sanketnagdive can you please re-deploy coKreat creatioin portal using following release tags
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC22

this will fix CO-950
CC @ashokreddy1208 @vinukumar-vs @santhosh-tg @gandham-santhosh

**Reply by sanketnagdive** [reply]
@shubhambansaltarento It is deployed

**Comment by shubhambansaltarento** [comment]
@sanketnagdive can you please re-deploy coKreat creatioin portal using following release tags
https://github.com/Sunbird-coKreat/creation-portal/releases/tag/release-7.0.0_RC23

this will fix CO-950
CC @ashokreddy1208 @vinukumar-vs @santhosh-tg @gandham-santhosh

**Reply by sanketnagdive** [reply]
@shubhambansaltarento It is deployed to cokreat staging

**Comment by ashokreddy1208** [comment]
Hi All,

This is the sign-off for R7.0.0 for Sunbird-coKreat as we have completed the new feature verification and Regression testing on the Dock Staging environment, Please find the detailed test case execution sheet [Here](https://docs.google.com/spreadsheets/d/1-0UaycSNdLqlXCmllYJEsl7xKgxXfxVx4uJYZYorSwI/edit?gid=518610797#gid=518610797&range=A1:O1)

Environment: Sunbird coKreat
Testing completion percentage: 100% Completed
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10652)

Scope | Status | Remarks
-- | -- | --
Scope delivered and signed off for the release (Functional + technical) | Done | Delivered Scope : [25 Items](https://docs.google.com/spreadsheets/d/1-0UaycSNdLqlXCmllYJEsl7xKgxXfxVx4uJYZYorSwI/edit?gid=1356983383#gid=1356983383&range=C1)
List of Tech items not tested by QA to be shared | Done | [12 Items](https://docs.google.com/spreadsheets/d/1-0UaycSNdLqlXCmllYJEsl7xKgxXfxVx4uJYZYorSwI/edit?gid=1533257667#gid=1533257667&range=F1)
List of items descoped | Done | NA
All defects to be in closed status (regression and new feature) | Done | All defects are fixed and verified by QA - [Link](https://project-sunbird.atlassian.net/issues/?jql=issuetype+in+%28Defect%29+AND+status+in+%28Assigned%2C+Closed%2C+Done%2C+%22Failed+Validation%22%2C+Fixed%2C+%22In+Development%22%2C+%22In+Progress%22%2C+%22In+Validation%22%2C+Invalid%2C+Open%2C+%22Pending+Deployment%22%2C+%22Ready+for+Development%22%2C+%22Ready+for+Release%22%2C+Reopen%2C+Reopened%2C+%22Selected+for+Contribution%22%2C+%22To+Do%22%2C+%22Validate%22%29+AND+affectedVersion+%3D+7.0.0+AND+created+%3E%3D+2024-01-11+AND+created+%3C%3D+2024-06-15+AND+labels+%21%3D+easyInstaller-7.0+AND+project+%3D+CO)
Call out all defects which have been converted to bugs | Done | [8 Items](https://project-sunbird.atlassian.net/issues/?jql=created%20%3E%3D%202024-01-11%0AAND%20created%20%3C%3D%202024-08-31%0AAND%20issuetype%20%3D%20Bug%0AAND%20status%20IN%20%28Open%2C%20Invalid%2C%20%22Selected%20for%20Contribution%22%2C%20%22In%20Validation%22%2C%20Done%2C%20DONE%2C%20%22In%20Development%22%2C%20%22Failed%20Validation%22%29%0AAND%20affectedversion%20%3D%207.0.0%0AAND%20project%20%3D%20CO%0AORDER%20BY%20created%20DESC)
Unit Test report for all New Features to be shared | Partially done | Respective TO's have signed off for their Building blocks, but haven't called out the exact % to QA
For an end user facing story, does it have developer documents in place | Done | [Release Notes](https://sunbird.gitbook.io/sunbird-cokreat-1/v/release-7.0.0-draft-1/use/release-notes)
Deployment tracker to be shared and signed off from DC | Done | [Installation](https://sunbird.gitbook.io/sunbird-cokreat-1/v/release-7.0.0-draft-1/use/getting-started/installation-document) / [upgrade document](https://sunbird.gitbook.io/sunbird-cokreat-1/v/release-7.0.0-draft-1/use/release-notes#cokreat-build-tags)

Also please find the Regression + New feature sign-off sheets from all the Building Blocks [here](https://docs.google.com/spreadsheets/d/1FHRVnT8aeRDjDsCxPUFHQAr1X3Lkqr0pYulYgkT5MVk/edit?gid=171383707#gid=171383707&range=D1).

<img width="968" alt="image" src="https://github.com/Sunbird-coKreat/Community/assets/58033030/60482926-bea7-42ef-9d92-8305806f0874">

CC :
Sunbird ED @vinukumar-vs @mohitga
Sunbird Knowlg @pallakartheekreddy @mohitga
Sunbird Inquiry @mohitga @krgauraw
Sunbird CoKreate @vinukumar-vs @mohitga
Sunbird Lern @rhwarrier @AmiableAnil
Sunbird Observ @rhwarrier @anandp504

DC's: @SanthoshVasabhaktula @vrayulu

Sunbird QA: @ashokreddy1208
@gandham-santhosh @santhosh-tg @sanketnagdive @kameshbhr
@madhucr @alok-os @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @kameshbhr @mmanimegalai @manju4201 @rajeevsathish @anisha147 @swayangjit @Hari-stackroute @Abhign [@kiranharidas187](https://github.com/kiranharidas187) [@baljeetsingh](https://github.com/baljeetsingh) @[siddhishinde0723](https://github.com/siddhishinde0723) [@parthlawate](https://ekstep.slack.com/team/U0HB3HJ73) [@manojLondhe](https://github.com/manojLondhe) [@swayangjit](https://github.com/swayangjit) [@vinod303](https://github.com/vinod303) @shubhambansaltarento
